README.md

```diff
@@ -80,9 +80,9 @@ You will create a shell script called `credentials.env` containing the following
 
 Be sure to grant oauth access to your cloud print account, as many of the tests will fail without it.
 
-You may also want to add the path to a custom configuration file which overrides settings from `defaults.config`:
+You may also want to add the path to a custom configuration file which overrides settings from `src/printapp/config.py`:
 
-    export PRINTAPP_SETTINGS=/path/to/printapp.config
+    export PRINTAPP_SETTINGS=~/calvinwebprint/secret-settings.py
```

Comment:
Since it's relative to `__init__.py`, then I think your original commit is actually better, because it still works even if you move the `calvinwebprint` directory.